javascript/static/dashboard/dashboard.html

```diff
@@ -0,0 +1,34 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gink Dashboard</title>
+    <link rel="stylesheet" href="/dashboard/styles.css">
+    <script type="text/javascript" src="/dashboard/packed.js"></script>
+    <script src="/dashboard/scripts/display_container.js"></script>
+    <script src="/dashboard/scripts/utility.js"></script>
+</head>
+
+<body>
+
+    <div id="container-contents">
+
+    </div>
+
+</body>
+<script>
+    document.addEventListener('DOMContentLoaded', async () => {
+        window.store = new gink.IndexedDbStore(true);
```

Comment:
The constructor the to the `IndexedDbStore` are the db name (string) and whether or not to reset the database (boolean).  I think you intended to use the default name and reset it, but in what you ended up doing was just using a database named "true".  In any case I would advise against resetting it for general dashboard use cases though perhaps allowing resetting for integration testing use cases.